spec/features/admin/orders/admin_order_index_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Order Index', type: :feature do
+  context 'as an admin' do
+    it 'should display all orders and correct info' do
+      admin = create(:admin)
+      user_w_order_1 = create(:user)
+      user_w_order_2 = create(:user)
+      user_w_order_3 = create(:user)
+      user_w_order_4 = create(:user)
+      order_1 = create(:packaged_order, user: user_w_order_1)
+      order_2 = create(:order, user: user_w_order_2)
+      order_3 = create(:shipped_order, user: user_w_order_3)
+      order_4 = create(:cancelled_order, user: user_w_order_4)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+      visit admin_dashboard_path
+
+      expect(page.all("div")[0]).to have_content(order_1.user.name)
```

Comment:
```suggestion
      expect(page).to have_link(order_1.user.name)
```
Similar for all (except they'll just be `have_content`)